nuxt.config.ts

```diff
@@ -34,6 +35,32 @@ export default defineNuxtConfig({
   routeRules: {
     '/api/search.json': { prerender: true },
   },
+  // Google Tag Manager
+  runtimeConfig: {
+    public: {
+      gtm: {
+        id: 'GTM-xxxxxx',
```

Comment:
I think that the real GTM ID and auth parameters should be provided.

Reply:
Yes, this is obtained from the personal profile on GTM.